docs/book/src/developer/tilt.md

```diff
@@ -82,6 +82,52 @@ base64 -i ~/path/to/gcp/credentials.json
 }
 ```
 
+{{#/tab }}
+{{#tab AZURE}}
+
+You would prepare an Azure service principal. Here is an [azure-cli example](https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli?view=azure-cli-latest):
+
+```bash
+# get the service principal name and subscription id
+SERVICE_PRINCIPAL_NAME=ServicePrincipalName 
+SUBSCRIPTION_ID=$(az account show --query id --output tsv)
+
+# set the subscription id when having multiple subscriptions
+az account set --subscription $SUBSCRIPTION_ID
+
+# get the tennant id
+TENNANT_ID=$( az account show --query tenantId --output tsv)
+
+# get the secret
+SECRET=$(az ad sp create-for-rbac --name http://$SERVICE_PRINCIPAL_NAME --query password --output tsv)
+
+# get the client id
+CLIENT_ID=$(az ad sp show --id http://$SERVICE_PRINCIPAL_NAME --query appId --output tsv)
+
+# check the result
```

Comment:
why do you need to echo the secret? if the instructions are correct this step can be skipped

Reply:
If SERVICE PRINCIPAL NAME is duplicated, no secret makes.